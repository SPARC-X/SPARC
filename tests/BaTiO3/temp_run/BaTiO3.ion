#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name> 
# PSEUDO_POT: <path/to/pseudopotential/file>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...

# Reminder: when changing number of atoms, change the RELAX flags accordingly
#           as well.

ATOM_TYPE: Ba                # atom type
PSEUDO_POT: Ba.psp8
N_TYPE_ATOM: 1               # number of atoms of this type
COORD:                       # coordinates follows
7.540463013371621   7.248833728543416   7.518607464589964

# 0	0	0

ATOM_TYPE: Ti                # atom type
PSEUDO_POT: Ti.psp8
N_TYPE_ATOM: 1               # number of atoms of this type
COORD:                       # coordinates follows
3.725768164762889   3.878328962888697   3.426961342823767

#0.5 0.5 0.5

ATOM_TYPE: O               # atom type 
PSEUDO_POT: O.psp8
N_TYPE_ATOM: 3               # number of atoms of this type
COORD:                       # coordinates follows
   4.052740031589397   7.190115020153727   3.856885000727702
   7.090600941523509   3.476460666072041   3.727807248988862
   3.601707961096307   3.648353082729004   0.199860166191001

#0   0.5 0.5
#0.5 0   0.5
#0.5 0.5 0

