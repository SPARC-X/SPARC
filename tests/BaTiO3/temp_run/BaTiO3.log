The program took 529.102 s.
