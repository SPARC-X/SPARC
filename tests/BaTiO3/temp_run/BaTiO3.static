***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Ba:
      0.9882651394       0.9500437390       0.9854007162
Fractional coordinates of Ti:
      0.4883051330       0.5082999951       0.4491430331
Fractional coordinates of O:
      0.5311585887       0.9423479712       0.5054895152
      0.9293054969       0.4556304936       0.4885723786
      0.4720456043       0.4781589886       0.0261939929
Total free energy (Ha): -1.313143011090600E+02
Atomic forces (Ha/Bohr):
 -4.0220671208E-03   1.4010380582E-03  -2.6109606218E-02
  9.2541382944E-02   1.8422722701E-01  -1.7742261484E-01
  1.2765161219E-02  -7.3620907309E-02   8.0674942889E-02
 -3.3933472207E-02  -8.2103827791E-02   5.8462671256E-02
 -6.7351004835E-02  -2.9903529967E-02   6.4394606912E-02
Stress (GPa): 
 -2.6034113285E+01   5.7598620046E+00  -3.0525986139E+00 
  5.7598620046E+00  -2.8132621607E+01   2.1127442616E+00 
 -3.0525986139E+00   2.1127442616E+00  -3.9380264516E+01
