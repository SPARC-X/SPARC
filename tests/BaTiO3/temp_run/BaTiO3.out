***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:22:43 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.63 7.63 7.63 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 76 76 76
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Gaussian
SMEARING: 0.007349864435
CHEB_DEGREE: 50
NSTATES: 29
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 1.01E-05
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: BaTiO3
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 1
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 1
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  1
Mesh spacing                       :  0.100395 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  BaTiO3.out
Total number of atom types         :  3
Total number of atoms              :  5
Total number of electrons          :  40
Atom type 1  (valence electrons)   :  Ba 10
Pseudopotential                    :  Ba.psp8
Pseudocharge radii of atom type 1  :  6.53 6.53 6.53 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  Ti 12
Pseudopotential                    :  Ti.psp8
Pseudocharge radii of atom type 2  :  6.53 6.53 6.53 (x, y, z dir)
Number of atoms of type 2          :  1
Atom type 3  (valence electrons)   :  O 6
Pseudopotential                    :  O.psp8
Pseudocharge radii of atom type 3  :  6.53 6.53 6.53 (x, y, z dir)
Number of atoms of type 3          :  3
Estimated total memory usage       :  676.52 MB
Estimated memory per processor     :  676.52 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.6225225058E+01        1.320E-01        88.637
2            -2.6265507900E+01        7.333E-02        28.505
3            -2.6265984130E+01        4.780E-02        28.390
4            -2.6266882919E+01        3.894E-02        28.671
5            -2.6263342483E+01        1.458E-02        28.319
6            -2.6262910422E+01        7.054E-03        27.728
7            -2.6262870349E+01        3.395E-03        27.904
8            -2.6262860449E+01        9.249E-04        27.506
9            -2.6262860147E+01        5.169E-04        27.457
10           -2.6262860163E+01        1.990E-04        26.982
11           -2.6262860215E+01        9.819E-05        26.193
12           -2.6262860192E+01        4.483E-05        26.088
13           -2.6262860188E+01        1.452E-05        25.967
14           -2.6262860189E+01        6.345E-06        25.544
15           -2.6262860216E+01        2.578E-06        25.475
16           -2.6262860218E+01        1.184E-06        24.619
17           -2.6262860182E+01        4.563E-07        20.028
Total number of SCF: 17    

k = [0.000, 0.000, 0.000]
Occupation of state 26 = 0.000000000000000.
Occupation of state 29 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.6262860222E+01 (Ha/atom)
Total free energy                  : -1.3131430111E+02 (Ha)
Band structure energy              : -1.1107917645E+01 (Ha)
Exchange correlation energy        : -2.1381021688E+01 (Ha)
Self and correction energy         : -2.0355748643E+02 (Ha)
Entropy*kb*T                       : -2.5685173180E-11 (Ha)
Fermi level                        :  2.3481747853E-01 (Ha)
RMS force                          :  1.2252510108E-01 (Ha/Bohr)
Maximum force                      :  2.7199699076E-01 (Ha/Bohr)
Time for force calculation         :  1.259 (sec)
Pressure                           :  3.1182333136E+01 (GPa)
Maximum stress                     :  3.9380264516E+01 (GPa)
Time for stress calculation        :  3.597 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  529.102 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
